libs/mavlink/include/mavlink/v1.0

```diff
@@ -1 +1 @@
-Subproject commit 9605f8087d3734ed89fa0aba6f343ced041fb3cb
+Subproject commit 39c14e81edf2eac7b5bfe6771eb56614f46ef098
```

Comment:
You need to update your mavlink version:
git submodule init
git submodule update